cluster/gce/trusty/master.yaml

```diff
@@ -213,6 +213,7 @@ script
 	set -o errexit
 	set -o nounset
 
+	. /etc/kube-configure-helper.sh
```

Comment:
I think what you mentioned is the health monitoring job, which is different from here.